src/Infrastructure.EntityFramework/Vault/Configurations/SecurityTaskEntityTypeConfiguration.cs

```diff
@@ -24,6 +24,18 @@ public void Configure(EntityTypeBuilder<SecurityTask> builder)
             .HasIndex(s => s.CipherId)
             .IsClustered(false);
 
+        builder
```

Comment:
ℹ️ We're trying to move top-level connections to `dbo.Organization` to the delete proc -- see one example at https://github.com/bitwarden/server/pull/5602/files#diff-23455d0afecf5d129c0181b1545efe42715cdaf3bff9a7f81a34bca7979e4f03. This is a mixed bag here so it's not strictly required, but we feel it's the "new way".